manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -91,6 +91,8 @@ configuration:
     # kube_iam_role: ""
     # log_s3_bucket: ""
     # wal_s3_bucket: ""
+    # wal_gs_bucket: ""
+    # gcp_credentials: ""
```

Comment:
Could you add these examples also in the configmap? But please with alphabetical sorting there.